source/common/formatter/substitution_formatter.cc

```diff
@@ -1918,6 +1926,67 @@ UpstreamHostMetadataFormatter::UpstreamHostMetadataFormatter(const std::string&
                           return host->metadata().get();
                         }) {}
 
+ProxyProtocolTlvsFormatter::ProxyProtocolTlvsFormatter(const std::string& tlv_type_str) {
+  // Specified tlv_type must be parsable as an int.
+  TRY_ASSERT_MAIN_THREAD {
```

Comment:
note: the Envoy linter requires using `TRY_ASSERT_MAIN_THREAD` instead of a raw try. 
```
vscode@ip-10-122-10-246:/workspaces/envoy-serverless$ ./tools/code_format/check_format.py fix && ./tools/code_format/format_python_tools.sh fix
Please note: `tools/code_format/check_format.py` no longer checks API `.proto` files, please use `tools/proto_format/proto_format.sh` if you are making changes to the API files
ERROR: From ./source/common/formatter/substitution_formatter.cc
ERROR: ./source/common/formatter/substitution_formatter.cc:1931: Don't use raw try, use TRY_ASSERT_MAIN_THREAD if on the main thread otherwise don't use exceptions.
```

Reply:
I see. That makes sense. I found `absl::SimpleAtoi` is used a lot instead. It doesn't throw exceptions. Here's an example: https://github.com/10gen/envoy-serverless/blob/0c17b0bd2f7fcc47408e806c223cf1e38a65050a/source/common/formatter/substitution_formatter.cc#L611

Reply:
Okay awesome, let's use that instead :) just pushed this change!